contracts/pool.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "/Users/aida/coding/hackmoney-poolio"
```

Comment:
What is the purpose of this file? Also, let's avoid using absolute paths.

Reply:
lmao